packages/components/modal/package.json

```diff
@@ -13,7 +13,14 @@
   },
   "license": "MIT",
   "scripts": {
-    "pika:build": "pika build"
+    "pika:build": "pika build",
+    "test": "karma start --coverage",
+    "test:watch": "karma start --auto-watch=true --single-run=false",
+    "test:update-snapshots": "karma start --update-snapshots",
+    "test:prune-snapshots": "karma start --prune-snapshots",
+    "test:legacy": "karma start --compatibility all --coverage",
+    "test:legacy:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage"
```

Comment:
Just the standard scripts. Decided not leave any out in case you make peace with snapshots